### PR TITLE
Make Django 1.9 compatible #ALTOS-9048

### PR DESCRIPTION
**Pull request by alexmadjar**
Makes django-scheduler compatible with Django 1.7, 1.8 and 1.9

### CODE REVIEW

File: schedule/models/calendars.py [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # -*- coding: utf-8 -*-
 import pytz
-from django.contrib.contenttypes import generic
+try:
+    from django.contrib.contenttypes.generic import GenericForeignKey
+except ImportError:
+    from django.contrib.contenttypes.fields import GenericForeignKey
```
Comment (ryochiji) [inline schedule/models/calendars.py]: According to the docs, this should work for both [1.7](https://docs.djangoproject.com/en/1.7/ref/contrib/contenttypes/#generic-relations) and [1.9](https://docs.djangoproject.com/en/1.9/ref/contrib/contenttypes/#generic-relations).

Comment (alexmadjar) [inline schedule/models/calendars.py]: an excellent point

Comment (alexmadjar) [inline schedule/models/calendars.py]: updated

**Comment by ryochiji** [comment]
👍

**Pull request merged**